packages/core/src/components/MonthPicker/MonthPicker.scss

```diff
@@ -0,0 +1,17 @@
+@import '@cmsgov/design-system-support/src/index';
+
+/*
+Month Picker
+
+@uswds https://standards.usa.gov/components/form-controls
```

Comment:
These links are meant to link to the related pattern in USWDS, but USWDS doesn't have a Month Picker component, so we can remove this.